lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($value instanceof OCI8Descriptor) {
```

Comment:
I would try to keep it simple then and map `PDO::PARAM_STR` to OCI descriptor + `OCI_B_CLOB` and `PDO::PARAM_LOB` to OCI descriptor + `OCI_BLOB` if that works for all cases. We should able to keep complete BC and do not need to touch other components of DBAL in this case. What do you think?